x/staking/keeper/params.go

```diff
@@ -30,18 +30,11 @@ func (k Keeper) MaxValidators(ctx sdk.Context) (res uint16) {
 	return
 }
 
-// BondDenom renturns  the param Bondable coin denomination
```

Comment:
just return sdk.DefaultBondDenom here to make low diff

Reply:
but i will bring so many useless stack call for an empty function